tests/test_fbx.py

```diff
@@ -79,3 +82,13 @@ def test_derivative(modelJ0023, toasJ0023):
             assert np.nanmax(relative_diff) < tol, msg
         else:
             continue
+
+
+def test_summary_FB():
+    m, t = get_model_and_toas(
+        os.path.join(datadir, parJ0023ell1), os.path.join(datadir, timJ0023)
+    )
+    f = fitter.WLSFitter(toas=t, model=m)
+
+    # Ensure print_summary runs without an exception for an ELL1 model with FBX
+    f.print_summary()
```

Comment:
Can you extract the summary to a string, and look in the string for appropriate text? Like, should it report a PB so you could look for PB in the string?

Reply:
I'm not sure I completely understand, but my original issue was just that `print_summary()` threw an exception when using an ELL1 binary model that contains FBX params. That is now fixed.

Reply:
(PB is reported in the summary.)